content/md/imports-exports/monitor-jobs.md

```diff
@@ -9,57 +9,91 @@ related: imports, exports, product-export-builder, discover-the-dashboard, what-
 
 # What is a job?
 
-You have several ways to follow the status of a job. A job can be:
-- an [import](imports.html),
-- an [export](exports.html),
-- a [mass edit](product-mass-actions.html),
-- a [rules](what-is-a-rule.html) execution _(EE only)_
-- a calculation of [project](what-is-a-project.html).
+Some actions trigger a back-end process called a "job". These actions can be:
+* an [import](imports.html),
+* an [export](exports.html),
+* a [mass action](product-mass-actions.html),
+* a calculation of [project](what-is-a-project.html) _(EE only)_
+* a [rule](what-is-a-rule.html) execution _(EE only)_
+* the approval/rejection of [proposals](proposals-workflow.html) _(EE only)_
+
+Here is what happens when you launch a job:
+
+1.  First, you are notified by a flash message when a job starts. Friendly warning, the approval/rejection of proposals don't trigger flash messages.
+![Jobs flash message](../img/Jobs_flash-message.png)
+1. Then, while the job is running, it will be listed in the Process Tracker amongst all the other jobs you launched. To access it, go to `Activity` and click on `Process tracker`.
+![List of jobs in process tracker](../img/Jobs_process-tracker.png)  
+Note that you can also check the status of your launched jobs on the dashboard. To access it, go to `Activity` and click on `Dashboard`.
+![List of jobs in dashboard](../img/Jobs_dashboard.png)
+1.  Finally, when the job is complete, a notification appears at the top of your screen. You can always go back to the job itself if you need more details.
+![Notification of a job complete](../img/Jobs_notification.png)
 
-# How to monitor?
```

Comment:
As you removed this title, can you make sure it's not referenced somewhere else in the help center? If it's the case, you'll need to change the link to another one in order to make it work again.

Reply:
Done, 1 correction made